src/utilities/util_verify.c

```diff
@@ -117,6 +124,17 @@ usage(void)
       "verify %s\n",
       progname, usage_prefix,
       "[-d dump_address | dump_blocks | dump_history | dump_layout | "
-      "dump_offsets=#,# | dump_pages] [-h] [-S] uri");
+      "dump_offsets=#,# | dump_pages] [-s] -a|uri");
+    (void)fprintf(stderr,
+      "Verify options:\n"
+      "\t"
+      "-a\t"
+      "Verify the entire history store, not applicable against a URI \n"
+      "\t"
+      "-d\t"
+      "Dump more information while verifying the URI \n"
+      "\t"
+      "-s\t"
+      "Verify the URI against the stable timestamp, valid only after a rollback-to-stable \n");
```

Comment:
This is new, no other `wt` command does this kind of expanded help screen as far as I know. Why is `verify` special?

Reply:
I thought it will be useful to explain what "-a" and "-s" are for. I agree we are inconsistent with other commands now, for instance:
```
$ ./wt -h WT_TEST/test_hs06.test_hs06.test_hs_instantiated_modify\(column\).0/ write
usage: wt [-LRrSVv] [-C config] [-E secretkey] [-h home] write [-ao] uri key ...
```
I do not know what that "-a" or "-o" in write command are for.

Even though we are not consistent with other wt commands, I will like to keep the additional help screen.

@agorrod , do you have an opinion around this comment ?

Reply:
If we're moving away from the WiredTiger documentation (where this stuff was historically described in more detail), then I agree we probably need more verbose descriptions.

Someone needs to take on the task of making all of the help messages consistent, it doesn't make sense to have only one command with a verbose help display.

Reply:
Alright, I have decided to revert the documentation changes to the wt utility and created WT-5661 to address self-documentation for all the commands.